DAB.md

```diff
@@ -1937,6 +1938,7 @@ versions | a list of versions of the protocol implemented on the device
 | LG ThinQ           |   LGThinQ               |
 | Yandex Alice       |   YandexAlice           |
 | TiVo Conversation  |   TiVoConversation      |
+| Whale  AI Voice    |   Zeasn ASR             |
```

Comment:
Is it possible to not have a space here?

Reply:
Sorry, I didn't understand what you mean. Can you explain it in more detail?

Reply:
At present, we are still making adjustments.